operator/app/pom.xml

```diff
@@ -35,7 +35,6 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <quarkus.operator.sdk.version>3.0.7</quarkus.operator.sdk.version>
-        <quarkus.version>2.7.5.Final</quarkus.version>
```

Comment:
Seems we're actually NOT overriding the Quarkus version used by the SDK:
```
mvn dependency:tree | grep quarkus
Picked up JAVA_TOOL_OPTIONS: -Djava.awt.headless=true
[INFO] +- io.quarkiverse.operatorsdk:quarkus-operator-sdk:jar:3.0.7:compile
[INFO] |  +- io.quarkiverse.operatorsdk:quarkus-operator-sdk-common:jar:3.0.7:compile
[INFO] |  +- io.quarkus:quarkus-arc:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-core:jar:2.7.5.Final:compile
[INFO] |  |  +- io.quarkus:quarkus-ide-launcher:jar:2.7.5.Final:compile
[INFO] |  |  +- io.quarkus:quarkus-development-mode-spi:jar:2.7.5.Final:compile
[INFO] |  |  +- io.quarkus:quarkus-bootstrap-runner:jar:2.7.5.Final:compile
[INFO] |  |  \- io.quarkus:quarkus-fs-util:jar:0.0.9:compile
[INFO] |  +- io.quarkus.arc:arc:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-openshift-client:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-kubernetes:jar:2.7.5.Final:compile
[INFO] |  |  \- io.quarkus:quarkus-container-image:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-container-image-jib:jar:2.7.5.Final:compile
[INFO] |  \- io.quarkus:quarkus-smallrye-health:jar:2.7.5.Final:compile
[INFO] |     +- io.quarkus:quarkus-vertx-http:jar:2.7.5.Final:compile
[INFO] |     |  +- io.quarkus:quarkus-security-runtime-spi:jar:2.7.5.Final:compile
[INFO] |     |  +- io.quarkus:quarkus-mutiny:jar:2.7.5.Final:compile
[INFO] |     |  |  +- io.quarkus:quarkus-smallrye-context-propagation:jar:2.7.5.Final:compile
[INFO] |     |  +- io.quarkus:quarkus-vertx-http-dev-console-runtime-spi:jar:2.7.5.Final:compile
[INFO] |     |  +- io.quarkus.security:quarkus-security:jar:1.1.4.Final:compile
[INFO] |     |  +- io.quarkus:quarkus-vertx:jar:2.7.5.Final:compile
[INFO] |     |  |  +- io.quarkus:quarkus-netty:jar:2.7.5.Final:compile
[INFO] |     \- io.quarkus:quarkus-jsonp:jar:2.7.5.Final:compile
[INFO] +- io.quarkiverse.operatorsdk:quarkus-operator-sdk-csv-generator:jar:3.0.7:compile
[INFO] +- io.quarkus:quarkus-resteasy-jackson:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-resteasy:jar:2.7.5.Final:compile
[INFO] |  |  \- io.quarkus:quarkus-resteasy-server-common:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-jackson:jar:2.7.5.Final:compile
[INFO] +- io.quarkus:quarkus-rest-client:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-resteasy-common:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-apache-httpclient:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-rest-client-config:jar:2.7.5.Final:compile
[INFO] +- io.quarkus:quarkus-rest-client-jackson:jar:2.7.5.Final:compile
[INFO] +- io.quarkus:quarkus-openshift:jar:2.7.5.Final:compile
[INFO] |  +- io.quarkus:quarkus-kubernetes-client-internal:jar:2.7.5.Final:compile
[INFO] |  \- io.quarkus:quarkus-container-image-openshift:jar:2.7.5.Final:compile
[INFO] +- io.quarkus:quarkus-minikube:jar:2.7.5.Final:compile
[INFO] +- io.quarkus:quarkus-kubernetes-client:jar:2.7.5.Final:compile
[INFO] +- io.quarkiverse.operatorsdk:quarkus-operator-sdk-csv-generator-deployment:jar:3.0.7:provided
[INFO] |  +- io.quarkiverse.operatorsdk:quarkus-operator-sdk-deployment:jar:3.0.7:provided
[INFO] |  |  +- io.quarkus:quarkus-arc-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  +- io.quarkus:quarkus-vertx-http-dev-console-spi:jar:2.7.5.Final:provided
[INFO] |  |  |  \- io.quarkus.arc:arc-processor:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-openshift-client-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  \- io.quarkus:quarkus-kubernetes-client-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |     \- io.quarkus:quarkus-jackson-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |        \- io.quarkus:quarkus-jackson-spi:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-container-image-jib-deployment:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-smallrye-health-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  +- io.quarkus:quarkus-vertx-http-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  |  +- io.quarkus:quarkus-vertx-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  |  |  \- io.quarkus:quarkus-netty-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  |  +- io.quarkus:quarkus-vertx-http-deployment-spi:jar:2.7.5.Final:provided
[INFO] |  |  |  |  +- io.quarkus:quarkus-mutiny-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  |  |  \- io.quarkus:quarkus-smallrye-context-propagation-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  |  \- io.quarkus.qute:qute-core:jar:2.7.5.Final:provided
[INFO] |  |  |  +- io.quarkus:quarkus-smallrye-health-spi:jar:2.7.5.Final:provided
[INFO] |  |  |  +- io.quarkus:quarkus-smallrye-openapi-spi:jar:2.7.5.Final:provided
[INFO] |  +- io.quarkus:quarkus-core-deployment:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus.gizmo:gizmo:jar:1.0.10.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-class-change-agent:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-devtools-utilities:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-builder:jar:2.7.5.Final:provided
[INFO] |  +- io.quarkus:quarkus-kubernetes-deployment:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-container-image-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  +- io.quarkus:quarkus-container-image-spi:jar:2.7.5.Final:provided
[INFO] |  |  |  \- io.quarkus:quarkus-container-image-util:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-kubernetes-spi:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-kubernetes-client-internal-deployment:jar:2.7.5.Final:provided
[INFO] |  |  |  \- io.quarkus:quarkus-kubernetes-client-spi:jar:2.7.5.Final:provided
[INFO] +- io.quarkus:quarkus-test-common:jar:2.7.5.Final:test
[INFO] |  +- io.quarkus:quarkus-jsonp-deployment:jar:2.7.5.Final:provided
[INFO] +- io.quarkus:quarkus-junit5:jar:2.7.5.Final:test
[INFO] |  +- io.quarkus:quarkus-bootstrap-core:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-bootstrap-app-model:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-bootstrap-maven-resolver:jar:2.7.5.Final:provided
[INFO] |  |  +- io.quarkus:quarkus-bootstrap-gradle-resolver:jar:2.7.5.Final:provided
[INFO] |  +- io.quarkus:quarkus-junit5-properties:jar:2.7.5.Final:test
```

Reply:
removed the operator changes as they only affected the mvn plugin, not the actual transient deps of the SDK. We'll wait for a new opSDK release that is built against 2.7.6, but that shouldn't affect this PR to go through.